.github/workflows/publish.yml

```diff
@@ -0,0 +1,52 @@
+name: publish
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, synchronize]
+
+concurrency:
+  group: preview-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  preview:
+    runs-on: ubuntu-latest
+    defaults:
+      run: 
+        working-directory: wordle-app
+    steps:
+      - name: 🏗 Setup repo
+        uses: actions/checkout@v2
+
+      - name: 🏗 Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+          cache: yarn
+          cache-dependency-path: wordle-app/yarn.lock
+
+      - name: 🏗 Setup Expo
+        uses: expo/expo-github-action@v7
+        with:
+          expo-version: latest
+          token: ${{ secrets.EXPO_TOKEN }}
+
+      - name: 📦 Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: 🚀 Publish production
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
+        run: expo publish --release-channel=production
```

Comment:
I just assumed the `production` release channel here, we can change this to anything we like. 

We can remove it to publish to `default` as well. We officially discourage it, because you can easily accidentally push on this branch, using `production` or `store` is more opt-in and therefore intentional.